fdbserver/tester.actor.cpp

```diff
@@ -2566,7 +2561,7 @@ void encryptionAtRestPlaintextMarkerCheck() {
 	bool success = true;
 	// Enumerate all files in the "simfdb/" folder and look for "marker" string
 	for (fs::recursive_directory_iterator itr(p); itr != end; ++itr) {
-		if (boost::filesystem::is_regular_file(itr->path())) {
+		if (fs::is_regular_file(itr->path())) {
 			std::ifstream f(itr->path().string().c_str());
 			if (f) {
```

Comment:
naming is bad. can we more explicit. Also do we really to c_str for creating a file. Can we explicit in check if the file is open or not. A btter name instead of f is current_file_stream

Reply:
Yeah, probably string() is also unnecessary for ifstream constructor.

About naming, do you mean the `f` here? Although I agree, the scope is also not too big. TBH, I am new in FDB code base, so I try to keep the changes minimal to the functionality, instead of refactoring in bigger chunks. That's why I didn't go beyond the issue much. It is also easier for the reviewer I think to keep the scope to one thing.